src/Card.tsx

```diff
@@ -4,6 +4,7 @@ export default function Card({ children }: { children: ReactNode }) {
   return (
     <div className="support__point">
       <img className="sp__image" />
+      <foo></foo>
```

Comment:
Is this needed?

Reply:
No. I think I might have accidentally committed that to test whether the build would have committed that or not.